tox.ini

```diff
@@ -45,7 +44,7 @@ deps = -r{toxinidir}/test-requirements.txt
 
 [testenv:py26]
 deps = -r{toxinidir}/test-requirements.txt
-commands = nosetests {posargs:tests/unittests cloudinit}
+commands = {envpython} -m pytest {posargs:tests/unittests cloudinit}
```

Comment:
Rather than making changes we should drop the sections for Py versions we no longer want to support.